stable/jfrog-platform/Chart.lock

```diff
@@ -4,27 +4,27 @@ dependencies:
   version: 10.3.18
 - name: rabbitmq
   repository: https://charts.jfrog.io/
-  version: 8.31.6
+  version: 11.9.3
```

Comment:
how do we do a minor upgrade on jfrog chart when there is a major upgrade on one of the charts dependencies?

We deployed this chart to production and start failing with authentication. It always worked. only on 10.12.1 jfrog-platform chart that it starts failing.

Reply:
I assume in fresh new installations when there is no PV available, it would work. Can you please test upgrade from 10.12.0 to 10.12.1? Xray will fail

Reply:
@EvertonSA We align with artifactory (minor) releases for platform chart . Can you please share logs to investigate further 
cc @Logeshwarsn